### DISCUSSION TITLE
Question about how one-time 100% charge works and its limitations

### DISCUSSION THREAD

**Discussion by Kowken** [seerge/g-helper#1502] [Q&A]
I have a question regarding that. So if I understand correctly, this one-time 100% won't take effect if one turns their laptop off right after setting it as it is not saved in the config right ? At least for me it does not (put in on charge at 60% with a 80% power limit and pushed the 100% before turning it off: turned it on at 80% the next morning).
I am also asking this question to see if I have a problem on my end or if saud behaviour is deemed normal.

Thanks in advance for your answers and have a great day.
Best regards.

**Comment by seerge** [comment]
@Kowken hello,

Correct, it will forget the one-time 100% charge setting on reboot (i.e. app restart)
